app-misc/lcd4linux/lcd4linux-0.11.0_pre20170527.ebuild

```diff
@@ -0,0 +1,201 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit autotools flag-o-matic python-single-r1
+
+DESCRIPTION="A small program that grabs information and displays it on an external LCD"
+HOMEPAGE="https://lcd4linux.bulix.org/"
+SRC_URI="https://www.bl4ckb0x.de/files/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE="dmalloc outb"
+REQUIRED_USE="lcd_devices_hd44780? ( !lcd_devices_hd44780-i2c )
+		lcd_devices_hd44780-i2c? ( !lcd_devices_hd44780 )
+		python? ( ${PYTHON_REQUIRED_USE} )"
+
+# Define the list of valid lcd devices.
+# Some drivers were removed from this list:
+# - lcdlinux: It's an ancient driver, which needs app-misc/lcd-linux, that made it never to the portage tree.
+# - lcdlinux: Besides, app-misc/lcd-linux won't compile on a modern linux kernel.
+# - st2205: It's needs dev-libs/libst2205,  which made it never to the portage tree and is quite outdated.
+IUSE_LCD_DEVICES=( astusb beckmannegle bwct crystalfontz curses cwlinux d4d dpf
+			ea232graphic efn futabavfd fw8888 g15 glcd2usb hd44780 hd44780-i2c
+			irlcd lcd2usb lcdterm ledmatrix lph7508 luise
+			lwabp m50530 matrixorbital matrixorbitalgx mdm166a milfordinstruments
+			newhaven noritake null pertelian phanderson
+			picgraphic picolcd picolcdgraphic png ppm routerboard
+			sample samsungspf serdisplib shuttlevfd simplelcd t6963
+			teaklcm trefon ula200 usbhub usblcd vnc wincornixdorf x11 )
+
+# Define the list of valid lcd4linux plugins.
+# Some plugins were removed from this list:
+# - imon: Uses telmond, which is only available on a fli4l router or an eisfair server.
+# - ppp: It has been replaced by the netdev plugin.
+# - seti: SETI@home software was replaced by sci-misc/boinc, which is not compatible.
+# - xmms: XMMS software was replaced by media-sound/xmms2, which is not compatible.
+IUSE_LCD4LINUX_PLUGINS=( apm asterisk button_exec cpuinfo dbus diskstats dvb exec event
+			fifo file gps hddtemp huawei i2c_sensors iconv isdn kvv
+			loadavg meminfo mpd mpris_dbus mysql netdev netinfo pop3
+			proc_stat python qnaplog raspi sample statfs uname uptime
+			w1retap wireless )
+
+# Add supported drivers from 'IUSE_LCD_DEVICES' to 'IUSE' and 'REQUIRED_USE'
+REQUIRED_USE="${REQUIRED_USE} || ("
+for lcd_driver in "${IUSE_LCD_DEVICES[@]}"; do
+	IUSE="${IUSE} lcd_devices_${lcd_driver}"
```

Comment:
You can save a lot of lines and that loop using `$(printf 'lcd_devices_%s ' ${IUSE_LCD_DEVICES[@]})` (same for `IUSE_LCD4LINUX_PLUGINS` below)

Reply:
Thanks, changed those blocks now and pushed a new version.